packages/bpk-component-popover/src/BpkPopover.js

```diff
@@ -40,7 +42,19 @@ const bindEventSource = (source, callback) => event => {
   callback(event, { source });
 };
 
-const BpkPopover = props => {
+export type Props = {
+  id: string,
+  onClose: Function,
```

Comment:
Don't use `Function`. From the [docs](https://flow.org/en/docs/types/functions/)

> However, if you need to opt-out of the type checker, and don’t want to go all the way to any, you can instead use Function. Function is unsafe and should be avoided.

From the code it looks like the type should be 

```javascript
onClose: (source: $Values<EVENT_SOURCE>) => void,
```

Reply:
I ended up with:
```
onClose: (
    event: SyntheticEvent<>,
    props: {
      source: $Values<typeof EVENT_SOURCES>,
    },
  ) => void,
```

Reply:
👍

Reply:
I like this; it's more readable :)